convert_hf_to_gguf.py

```diff
@@ -702,6 +695,8 @@ def get_vocab_base_pre(self, tokenizer) -> str:
         if chkhsh == "ccc2ef013c104be7bae2965776d611e1d7a8a2a9c547dd93a682c9a9fc80352e":
             # ref: https://huggingface.co/Xenova/gpt-4o
             res = "gpt-4o"
+        if chkhsh == "a81863d07e75497e2194eb1a1574d5e5cd4d5f85a87a0728b922bf2bed6fb327":
+            res = "bert"
```

Comment:
The newly added tokenizer is `nomic-embed-text-v2-moe` and not `bert`, is this expected?

And also this list is auto-generated, please make sure not to modify it manually